<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IRiverPla</class>
 <widget class="QMainWindow" name="IRiverPla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRiverPla</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name for Playlist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPlaylistName"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="lstFiles">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DropOnly</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QGridLayout" name="gridLayout">
              <item row="8" column="0">
               <widget class="QPushButton" name="btnDown">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Move Down</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+Down</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnAdd">
                <property name="text">
                 <string>Add Files</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="btnGenerate">
                <property name="text">
                 <string>Generate PLA</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="cbKeepFolder">
                <property name="toolTip">
                 <string>Preserve 1 folder level when copying music to 'Music destination'. If not checked then all music files are copied without folder level.</string>
                </property>
                <property name="text">
                 <string>Preserve file folder</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+A</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="btnDestination">
                <property name="toolTip">
                 <string>Where you want to copy your music files</string>
                </property>
                <property name="text">
                 <string>Music destination</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="btnPlaylistdestination">
                <property name="toolTip">
                 <string>Where you want the PLA playlist file to be copied</string>
                </property>
                <property name="text">
                 <string>Playlist destination</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="btnUp">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Move Up</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+Up</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="btnRemove">
                <property name="text">
                 <string>Remove selected</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextEdit" name="edtLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="documentTitle">
           <string>PLA format</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Add_to_playlist"/>
    <addaction name="actionRemove"/>
    <addaction name="action_Destination"/>
    <addaction name="actionMusic_destination"/>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuA_bout">
    <property name="title">
     <string>A&amp;bout</string>
    </property>
    <addaction name="actionShow_Log"/>
    <addaction name="actionIriver_Plus"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuA_bout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Add_to_playlist">
   <property name="text">
    <string>Add to playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Destination">
   <property name="text">
    <string>Playlist Destination</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionIriver_Plus">
   <property name="text">
    <string>iriver&amp;Plus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionShow_Log">
   <property name="text">
    <string>Show Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionMusic_destination">
   <property name="text">
    <string>Music destination</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>IRiverPla</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
